.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+If updating the specs in /openapi, you can test the changes by merging your branch into integration, and navigating to https://open-payments-integration.readme.io
+-->
```

Comment:
@wilsonianb There wasn't an easy way to make this "dynamic" where we add this blurb only if there are changes in `/openapi`, so just kept it as such